src/internal/methods/NeoEspBitBangMethod.cpp

```diff
@@ -137,19 +149,44 @@ void IRAM_ATTR neoEspBitBangWriteSpacingPixels(const uint8_t* pixels,
             subpix = *pixels++;
 
             // if pixel spacing is needed
-            if (tSpacing)
+            element++;
+            if (element == sizePixel)
             {
-                element++;
-                if (element == sizePixel)
+                element = 0;
+
+                // Hack: permit interrupts to fire
+                // If we get held up more than the latch period, restart.
+                // We do this at the end of an element to ensure that each element always gets *some* valid 
+#if !defined(ARDUINO_ARCH_ESP32)
+                interrupts();
+#endif                
+                // wait for pixel spacing
+                if (tSpacing)
                 {
-                    element = 0;
-
-                    // wait for pixel spacing
                     while ((getCycleCount() - cyclesNext) < tSpacing);
                 }
+#if !defined(ARDUINO_ARCH_ESP32)
+                noInterrupts();
+#endif
+                if ((getCycleCount() - cyclesNext) > tReset) {
+                    if (++retries > 8) break;   // give up
```

Comment:
using a break in an if should only be used if improves the readability of the code.  Since this is a simple, it should be placed into the if above and remove the break;